selenium_tests/course_info/course_info_add_people_tests.py

```diff
@@ -2,52 +2,52 @@
 
 from selenium_common.base_test_case import get_xl_data
 from selenium_tests.course_info.course_info_base_test_case \
-    import TEST_USERS_WITH_ROLES_PATH
+    import TEST_USER_SEARCH_TERMS_PATH
+
 from selenium_tests.course_info.course_info_base_test_case \
     import CourseInfoBaseTestCase
 
 
 @ddt
 class AddPeopleTests(CourseInfoBaseTestCase):
 
-    @data(*get_xl_data(TEST_USERS_WITH_ROLES_PATH))
+    @data(*get_xl_data(TEST_USER_SEARCH_TERMS_PATH))
     @unpack
-    def test_add_person(self, test_case_id, test_user, canvas_role, role_id):
+    def test_add_people(self, test_case_id, search_terms, canvas_role, role_id,
+                        successes, failures):
         """ verify the person search and add functionality """
 
         # Note: ICOMMONS_REST_API_HOST environment needs to match the LTI tool
         # environment (because of shared cache interactions)
 
-        # ensure person is not in course before attempting to add using API;
-        # remove ALL roles/enrollments for the test user in this course
+        # ensure people are not in course before attempting to add using API;
+        # remove ALL roles/enrollments for the test users in this course
         # to ensure no incidental data causes conflict when we try to add
-        self.api.remove_user(self.test_settings['test_course']['cid'],
-                             test_user)
 
-        self._load_test_course()
+        test_course_key = 'test_course_SB_ILE'
```

Comment:
Now that you've [changed the test_course](https://github.com/Harvard-University-iCommons/canvas_account_admin_tools/pull/104/files#diff-161fca96bd4a31782cda875c287a0601R37) do we just want to be using that, instead of `test_course_SB_ILE`?